src/main/java/org/generationcp/middleware/service/api/study/germplasm/source/GermplasmStudySourceRequest.java

```diff
@@ -0,0 +1,222 @@
+package org.generationcp.middleware.service.api.study.germplasm.source;
+
+import org.generationcp.middleware.pojos.SortedPageRequest;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@AutoProperty
+public class GermplasmStudySourceRequest {
```

Comment:
I think to rename this class to GermplasmStudySourceSearchRequest will make it more clear